src/HavenoClient.test.ts

```diff
@@ -2217,6 +2217,31 @@ test("Selects arbitrators which are online, registered, and least used", async (
   }
 });
 
+test("Can get trade statistics", async () => {
+  // a simple test to exercise the trade statistics API
+  const tradeStatisticsArbitrator = await arbitrator.getTradeStatistics();
+  const tradeStatisticsUser1 = await user1.getTradeStatistics();
+  const tradeStatisticsUser2 = await user2.getTradeStatistics();
+  HavenoUtils.log(0, "Trade statistics size (arb/u1/u2): " + tradeStatisticsArbitrator.length + "/" + tradeStatisticsUser1.length + "/" + tradeStatisticsUser2.length);
+  assert(tradeStatisticsArbitrator.length === tradeStatisticsUser1.length && tradeStatisticsUser1.length === tradeStatisticsUser2.length);
+});
+
+test("Check one trade statistic written per trade", async () => {
```

Comment:
Thanks. Let's move this test into "Can complete a trade within a range", to avoid creating another test which executes a single trade, since the tests already take ~52 minutes to complete. Logically testing the trade stats is part of testing a single trade, too.